clang/lib/DPCT/Error.cpp

```diff
@@ -28,8 +28,8 @@ void ShowStatus(int Status, std::string Message) {
   case MigrationSkipped:
     StatusString = "Some migration rules were skipped.";
     break;
-  case MigrationCmakeScriptCompleted:
-    StatusString = "Migration of CMake script completed.";
+  case MigrationBuildScriptCompleted:
+    StatusString = "Migration of CMake/ Python setup script completed.";
```

Comment:
### 
```suggestion
    StatusString = "Migration of CMake or Python setup script completed.";
```

Reply:
Changed to 

"Migration of build script(s) completed."